swiftnav/src/reference_frame/mod.rs

```diff
@@ -163,37 +192,86 @@ pub enum ReferenceFrame {
 /// Where $p$ is the constant value, $\dot{p}$ is the rate of
 /// change, and $\tau$ is the reference epoch.
 ///
+/// # Parameter Units
+///
+/// Input parameters are stored in standard geodetic units:
+/// - **Translation** (`tx`, `ty`, `tz`): millimeters (mm)  
+/// - **Translation rates** (`tx_dot`, `ty_dot`, `tz_dot`): mm/year
+/// - **Scale** (`s`): parts per billion (ppb)
+/// - **Scale rate** (`s_dot`): ppb/year  
+/// - **Rotation** (`rx`, `ry`, `rz`): milliarcseconds (mas)
+/// - **Rotation rates** (`rx_dot`, `ry_dot`, `rz_dot`): mas/year
+/// - **Reference epoch** (`epoch`): decimal years
+///
+/// These units are automatically converted to SI units during computation.
+///
+/// # Sign Convention
+///
 /// There are several sign conventions in use for the rotation
 /// parameters in Helmert transformations. In this implementation
 /// we follow the IERS conventions, which is opposite of the original
 /// formulation of the Helmert transformation.
 #[derive(Debug, PartialEq, PartialOrd, Clone, Copy)]
 pub struct TimeDependentHelmertParams {
-    t: Vector3<f64>,
-    t_dot: Vector3<f64>,
-    s: f64,
-    s_dot: f64,
-    r: Vector3<f64>,
-    r_dot: Vector3<f64>,
-    epoch: f64,
+    pub t: Vector3<f64>,
+    pub t_dot: Vector3<f64>,
+    pub s: f64,
+    pub s_dot: f64,
+    pub r: Vector3<f64>,
+    pub r_dot: Vector3<f64>,
+    pub epoch: f64,
 }
 
 impl TimeDependentHelmertParams {
-    const TRANSLATE_SCALE: f64 = 1.0e-3;
-    const SCALE_SCALE: f64 = 1.0e-9;
-    const ROTATE_SCALE: f64 = (std::f64::consts::PI / 180.0) * (0.001 / 3600.0);
+    /// Scale factor for translation parameters (converts mm to m)
+    pub const TRANSLATE_SCALE: f64 = 1.0e-3;
+    /// Scale factor for scale parameters (converts ppb to unit scale)
+    pub const SCALE_SCALE: f64 = 1.0e-9;
+    /// Scale factor for rotation parameters (converts milliarcseconds to radians)
+    pub const ROTATE_SCALE: f64 = (std::f64::consts::PI / 180.0) * (0.001 / 3600.0);
+
+    /// Create a [`TimeDependentHelmertParams`] object with zero in every field.
+    ///
+    /// # Note
+    ///
+    /// The [`TimeDependentHelmertParams::epoch`] field needs to represent a real time, and an
+    /// epoch of `0.0` is almost certainly not what you intend. This is best used to initialize
+    /// all other fields to zero like this:
+    ///
+    /// ```rust
+    /// # use swiftnav::reference_frame::TimeDependentHelmertParams;
+    /// let params = TimeDependentHelmertParams {
+    ///   epoch: 2020.0,
+    ///   ..TimeDependentHelmertParams::zeros()
+    /// };
+    /// ```
+    #[must_use]
+    pub const fn zeros() -> TimeDependentHelmertParams {
+        TimeDependentHelmertParams {
+            t: Vector3::new(0.0, 0.0, 0.0),
+            t_dot: Vector3::new(0.0, 0.0, 0.0),
+            s: 0.0,
+            s_dot: 0.0,
+            r: Vector3::new(0.0, 0.0, 0.0),
+            r_dot: Vector3::new(0.0, 0.0, 0.0),
+            epoch: 0.0,
+        }
+    }
```

Comment:
Is there any reason to not just make this the default? This seems like a reasonable default but I am not entirely sure that it makes sense to be either. If you agree it might be good to do a default impl on this struct